pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -255,8 +263,18 @@ func desiredLoadBalancerService(ci *operatorv1.IngressController, deploymentRef
 				// NLBs require a different health check interval than CLBs
 				service.Annotations[awsLBHealthCheckIntervalAnnotation] = awsLBHealthCheckIntervalNLB
 			} else {
+				var additionalTags []string
+				if platformSpec != nil && platformSpec.AWS != nil {
+					for _, userTag := range platformSpec.AWS.UserTags {
+						if len(userTag.Key) > 0 {
+							additionalTags = append(additionalTags, userTag.Key+"="+userTag.Value)
+						}
+					}
+				}
+				service.Annotations[awsLBAdditionalResourceTags] = strings.Join(additionalTags, ",")
```

Comment:
Does it make sense to set the annotation if `additionalTags` is empty?

Reply:
> Does it make sense to set the annotation if `additionalTags` is empty?

I was wondering about that...

If you previously set "a=b,c=d" and this time around there are no tags then setting an empty value doesn't help here because it doesn't mean that "a=b,c=d" will be deleted.

I will change this to not set the annotation if `additionalTags` is empty.

Reply:
This still sets the annotation if `platform.AWS.ResourceTags` is nonempty but every entry has an empty key, right?  Maybe that edge-case isn't worth worrying about.